api/src/asyncIterableKit.js

```diff
@@ -60,15 +61,17 @@ export const makeTimerAsyncIterableKit = async (timer, delay, interval) => {
   const { notifier, updater } = makeNotifierKit();
   const repeater = E(timer).createRepeater(delay, interval);
 
-  /** @type {TimerWaker} */
-  await E(repeater).schedule({
-    // FIXME: It's a limit of the current API that we have no way to detect when a
-    // repeater has been disabled from within a handler.
+  const waker = Far('waker', {
+    // FIXME: It's a limit of the current API that we have no way to detect when
+    // a repeater has been disabled from within a handler.
     wake(timestamp) {
       updater.updateState(timestamp);
     },
   });
 
+  /** @type {TimerWaker} */
```

Comment:
Does this declaration bind to the correct object?

`repeater` is a `TimerRepeater`.  `.schedule()` returns a `Timestamp`.

`waker`, above, is a `TimerWaker`: it has a `wake()` method.

Reply:
Fixed